server/src/index.js

```diff
@@ -42,3 +43,5 @@ export const server = app.listen(PORT, (err) => {
 export const socketPath = '/sockets/';
 export const io = socketIO(server, { path: socketPath });
 export const db = mongoose.connection;
+
+populateWorkspaceAsNamespacesInIO();
```

Comment:
Call this after server starts.

Reply:
@nimish-gupta, called it in the `startServer` function.